angular/Grid/Getting-Started.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Getting Started for Angular-2 Grid
-description: How to create the Grid, data bind, enable paging, grouping, filtering and add summaries
+title: Getting Started for Angular-2 Grid | Syncfusion 
+description: Learn here more about getting started with Syncfusion Essential Angular Grid Control, its elements, and more.
 platform: Angular
 control: Grid
 documentation: ug
 keywords: ejGrid, Grid, Grid widget
 api: /api/Angular/grid
 ---
 
-# Getting started
+# Getting Started for Angular-2 Grid
```

Comment:
Getting Started for Angular Grid - change like this

Reply:
Done